src/style/app.module.css

```diff
@@ -498,7 +492,11 @@
 }
 
 .inputField:focus {
-  box-shadow: 0 0 0 0.25rem var(--light-blue);
+  border: 0.1px solid var(--dimp-white) !important;
+  background-color: var(--bs-white) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`